tool/parse_cli.c

```diff
@@ -110,79 +98,35 @@ void parse_genx509cert_cli(int argc, char **argv, struct cli_params *params)
 }
 
 
-static
-void parse_wrapkeys_cli(int argc, char **argv, struct cli_params *params)
-{
-    params->privkey = "priv.bin";
-    params->pubkey = "pub.bin";
-    params->asn1 = "priv.asn1.bin";
-    const char *usage_str = "Wrap keys in ASN.1 formatting.\n\n"
-        "Usage: %s %s [-h] [-v <file>] [-b <file>] [-a <file>]\n"
-        "\tOptions:\n"
-        "\t\t-h --help              Display this message.\n"
-        "\t\t-v --priv              Private key input location [default = priv.bin]\n"
-        "\t\t-b --pub               Public key input location [default = pub.bin]\n"
-        "\t\t-a --out-asn1          ASN.1 output location [default = priv.asn1.bin]\n"
-        ;
-
-    static struct option cli_options[] =
-    {
-        {"in-priv", required_argument, NULL, 'v'},
-        {"in-pub", required_argument, NULL, 'b'},
-        {"out-asn1", required_argument, NULL, 'a'},
-        {"help", no_argument, NULL, 'h'},
-        {NULL, 0, NULL, 0}
-    };
-
-    int c;
-    while ((c = getopt_long(argc, argv, "v:b:d:a:h", cli_options, NULL)) != -1) {
-        switch (c) {
-            case 'v':
-                params->privkey = optarg;
-                break;
-            case 'b':
-                params->pubkey = optarg;
-                break;
-            case 'a':
-                params->asn1 = optarg;
-                break;
-            case 'h':
-                printf(usage_str, argv[0], argv[1]);
-                exit(1);
-        }
-    }
-}
-
-
 static
 void parse_genroot_cli(int argc, char **argv, struct cli_params *params)
 {
-    params->pubkey = "root_pub.bin";
+    params->keypair = "root_keypair.bin";
```

Comment:
Can we change this default to something like `root_keys.asn1.bin`? Then, all the other places this is used will have to be changed...